backend/apps/sandbox/migrations/0015_maintainerscenario_maintainerevaluation.py

```
@@ -0,0 +1,48 @@
+import django.db.models.deletion
+import uuid
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sandbox', '0014_workspacesnapshot_snapshotfile_and_more'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)